test/e2e/specs/editor/editor__page-basic-flow.ts

```diff
@@ -26,7 +26,7 @@ describe( DataHelper.createSuiteTitle( 'Editor: Basic Post Flow' ), function ()
 	let editorIframe: Frame;
 	let pagesPage: PagesPage;
 	let editorSettingsSidebarComponent: EditorSettingsSidebarComponent;
-	let publishedUrl: string;
+	let publishedUrl: URL;
```

Comment:
Everything after this point are the specs being updated to work with the new return value of type `URL`.